### ISSUE TITLE
Invalid https://paperswithcode.com/media/about/evaluation-tables.json.gz linked from README

### ISSUE THREAD

**Issue by SergioSVQ** [paperswithcode/paperswithcode-data#18]
Hi all,

We are having problems trying to download the evaluation-tables.json.gz file (https://paperswithcode.com/media/about/evaluation-tables.json.gz ) since few days ago. Could someone please clear up what happens with that file ?We download updates regularly in the interest of assessing the data.

Thanks for sharing the data and I hope to hear from you soon. 

Regards

**Comment by rstojnic** [comment]
Hello!

These old URLs are now redirecting to the new location, so it should be all working as long as your client can follow HTTP redirects. However, just in case I've also updated the links to directly point to new location in 9e752b1f8f8deebbc88bb6a2f7f5554dd5afd96a

**Comment by SergioSVQ** [comment]
Many thanks in advance for you quick answer . The fact is that get to reach to the URL but now I understand that the JSON has been changed. Let me explain. Until a few days ago, I downloaded the GZ file and had a process that unzipped and extracted the data for the sake of evaluating the data. The thing is that as soon as we start having problems downloading the GZ file, we can't use our process because when I reach to the URL, I see directly a JSON code (it's weird because the file name is evaluation-tables.json.gz, so it should be a gz file). We therefore have two questions:
1- Has the JSON code been modified recently?
2- Should I see the JSON code when I get to the https://paperswithcode.com/media/about/evaluation-tables.json.gz file instead of a GZ file as We got a few weeks ago?

@rstojnic 

Many thanks in advance.
